### PR TITLE
feat: add create/get asset

### PR DESCRIPTION
**Pull request by alkasm**

### CODE REVIEW

File: nominal/core/asset.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        self,
+        *,
+        name: str | None = None,
+        start: datetime | IntegralNanosecondsUTC | None = None,
+        end: datetime | IntegralNanosecondsUTC | None = None,
+        description: str | None = None,
+        properties: Mapping[str, str] | None = None,
+        labels: Sequence[str] | None = None,
+    ) -> Self:
+        """Replace run metadata.
```
Comment (alkasm) [inline nominal/core/asset.py:40]: this docstring references run in a couple places (copied from there)

File: nominal/__init__.py [hunk 2]
```diff
@@ -11,6 +12,7 @@
     LogSet,
     NominalClient,
     Run,
+    User,
```
Comment (alkasm) [inline nominal/__init__.py:15]: this was left out previously on accident!

File: nominal/core/asset.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    _clients: _Clients = field(repr=False)
+
+    class _Clients(HasAuthHeader, Protocol):
+        @property
+        def assets(self) -> scout_assets.AssetService: ...
+
+    def update(
+        self,
+        *,
+        name: str | None = None,
```
Comment (ad-nmnl) [inline nominal/core/asset.py:33]: Should this mirror the underlying API and be called `title`?

File: nominal/core/client.py [hunk 4]
```diff
@@ -563,6 +566,36 @@ def create_video_from_mcap_io(
...
+        labels: Sequence[str] = (),
+    ) -> Asset:
+        """Create an asset."""
+        request = scout_asset_api.CreateAssetRequest(
+            description=description,
+            labels=list(labels),
+            properties={} if properties is None else dict(properties),
+            title=name,
+            attachments=[],
+            data_scopes=[],
```
Comment (ad-nmnl) [inline nominal/core/client.py:584]: This is the most important part of creating an asset as it defines the underlying data. Would it be possible to implement this functionality?

File: nominal/core/asset.py (hunk 3 above)
Comment (alkasm) [reply nominal/core/asset.py:33]: We use name and title interchangeably at the API layer, but made the decision to unify this into "name" in the client lib regardless of what the API is using.

File: nominal/core/client.py (hunk 4 above)
Comment (jackparmer) [reply nominal/core/client.py:584]: @ad-nmnl how do you imagine `data_scopes` usage here? Is this like `ref_name`?

**Pull request merged**